modules/layers/src/icon-layer/icon-manager.js

```diff
@@ -238,6 +240,11 @@ export default class IconManager {
     }
   }
 
+  get loaded() {
+    const requestedIcons = Object.keys(this._mapping).length;
+    return this._autoPacking && requestedIcons && this._loadedIcons === requestedIcons;
```

Comment:
I'd recommend tracking the number of pending requests rather than the number of fulfilled requests:

```js
this._pendingCount++;
loadSomething(...).finally(() => this._pendingCount--);
```

That way you don't need to know how to calculate the total number of requests here with `_autoPacking` and `_mapping`. The auto packing logic may change in the future and maintainers may forget to update this function, given that there are no tests. If requests are tracked where they start, it will be much more robust.